docs/layers/identity/centralized-terraform-access.mdx

```diff
@@ -36,21 +30,185 @@ assume the role in any account.
 
-### AWS SAML
+### Deploying both AWS IAM Identity Center and AWS SAML
+
+AWS IAM Identity Center and AWS SAML are not mutually exclusive solutions. They can be used together to provide a seamless login experience for users. Cloud Posse frequently deploys both in customer environments - AWS SAML to log in ourselves for the duration of the engagement and AWS IAM Identity Center for the customer to use and support. That way the customer can access AWS via their chosen Identity Provider (IdP), and we can use GSuite SAML to access the Identity account team directly. When an engagement is complete, we can easily decommission our access by removing the SAML connection.
+
+:::tip AWS IAM Identity Center or AWS SAML? Which do I choose?
+
+The vast majority of our customers prefer AWS IAM Identity Center (SSO). The convenience of a web console login is hard to beat. However, some customers prefer SAML for its simplicity and compatibility with existing systems. We support both methods, and you can choose the one that best fits your needs.
+
+:::
+
+
+```mermaid
+flowchart TB
+    subgraph "Complete Access Architecture"
+        subgraph "core-root"
+            subgraph sso["Identity Center"]
+                developers_team_access["IdentityDevelopersTeamAccess"]
+                devops_team_access["IdentityDevopsTeamAccess"]
+            end
+        end
+
+        subgraph "core-identity"
+            subgraph aws_teams["aws-teams"]
+                devops_teams["devops"]
+                developers_teams["developers"]
+                spacelift_teams["spacelift"]
+            end
+        end
+
+        subgraph "plat-prod"
+            subgraph aws_team_roles_prod["aws-team-roles"]
+                terraform_prod["terraform"]
+                planner_prod["planner"]
+            end
+        end
+
+        subgraph "plat-sandbox"
+            subgraph aws_team_roles_sandbox["aws-team-roles"]
+                terraform_sandbox["terraform"]
+            end
+        end
+
+        developers_team["Developer Team"] --> developers_team_access
+        devops_team["DevOps Team"] --> devops_team_access
+        developers_team_access --> developers_teams
+        devops_team_access --> devops_teams
+
+        g_suite["G Suite"] -->|SAML Federated Identity| devops_teams
+        spacelift["Spacelift"] --> spacelift_teams
+
+        developers_teams --> planner_prod
+        devops_teams --> terraform_prod
+        spacelift_teams --> terraform_prod
+
+        developers_teams --> terraform_sandbox
+        devops_teams --> terraform_sandbox
+        spacelift_teams --> terraform_sandbox
+    end
+```
+
+## FAQ
+
+### Why use AWS SAML?
 
 The `aws-saml` component provides SAML access for Admin users to connect to the Identity account admin role `aws-teams`
 without AWS IAM Identity Center (Successor to AWS Single Sign-On).
```

Comment:
SAML offers granular control over identity management with your IdP in AWS. With our `aws-saml` component, you can integrate multiple IdPs, and we explicitly provision every role, policy, and trust relationship needed. The `aws-saml` component allows users to assume any number of roles to access AWS accounts that are associated with their IdP groups.